Automattic/jetpack
# Boost: Fix switching static front page to posts page during c.css generation

## Proposed changes:

* Remove Boost's GET params for critical css generation and disable modules from query_vars list.
* Fix returning home url for blog posts provider.

### Other information:


## Jetpack product discussion



n/a

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






### Test that home page content changes

* Make sure **you're not** running this branch;
* Set up a static home page and a blog page;
* Open the home page with the `jb-generate-critical-css` parameter. For example - http://jetpack-boost.test/?jb-generate-critical-css=1713879406768
* The content of the page should change to the posts listing, instead of the static page without an admin bar.

### Test that home page content is the correct one

* Make sure **you are** running this branch;
* Set up a static home page and a blog page;
* Open the home page with the `jb-generate-critical-css` parameter. For example - http://jetpack-boost.test/?jb-generate-critical-css=1713879406768
* The content of the page should not change and the admin bar should disappear.
